scripts/alembic

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+# A wrapper around the alembic command-line script that configures it with:
+# 1. The Triage PYTHONPATH
+# 2. The Triage alembic.ini
+# 3. The Triage database.yaml path
+#
+# All arguments are passed through to the installed 'alembic' command.
+
+PYTHONPATH='../' alembic -c src/triage/component/results_schema/alembic.ini -x db_config_file=database.yaml $@
```

Comment:
Completely up to you, but I think it might be useful to go all-in with your pal, `argcmdr` 😉 

Rather than a scripts directory, since you already have `argcmdr` installed, you can just add a `manage.py` to the repository root, for example:

    import argparse
    from pathlib import Path
    
    from argcmdr import local, LocalRoot
    
    
    ROOT_PATH = Path(__file__).parent.resolve()


    class Triage(LocalRoot):
        pass
    
    
    @Triage.register
    @local('remainder', metavar='alembic arguments', nargs=argparse.REMAINDER)
    def alembic(context, args):
        return context.local['env'][
            'PYTHONPATH=' + ROOT_PATH / 'src',
            'alembic',
            '-c', ROOT_PATH / 'src' / 'triage' / 'component' / 'results_schema' / 'alembic.ini',
            '-x', 'db_config_file=database.yaml',
            args.remainder,
        ]

The `manage` command installed by `argcmdr` will automatically load any file named `manage.py` in the current working directory and parse the command hierarchy starting from the most likely base, (this case the "root" command).

Obviously this doesn't make a tremendous immediate difference; but, it provides a nice base from which to expand your repository-based management commands.

Rather than maintaining separate proper executable (Bash) scripts, you just add to that `manage` module. And rather than executing the command with say `./scripts/alembic ...`, you'd say:

    manage alembic ...

The greatest immediate gain is, say, that you provide `manage -h` and, even, `manage [-d|--dry-run] alembic ...`.

Reply:
(Of course, since you also provide a `$ triage` command, you might like to name the root command class something like `Development`? It really makes almost no difference; it's just what will be printed when the user executes `manage --help`.)

Reply:
Excellent! Done.

Reply:
:+1: